tests/cluster/eh/test_eh_saving.py

```diff
@@ -93,3 +91,63 @@ def test_04_read_eh_capture_extractor(self):
             )
         )
         self.assertTrue(df.count(), 2)
+
+    def test_05_eh_json_orchestrator(self):
+        # the orchestrator has a complex functionality that can only be fully tested
+        # on a substantial holding of capture files. That is not possible here, but
+        # such tests were carried out during development.
+        # The situation here only tests the basic functions.
+
+        # Part 1, YMD partitioned
+        tc = TableConfigurator()
+        tc.register("CpTblYMD", {"name": "CaptureTableYMD"})
+        Spark.get().sql(
+            """
+            CREATE TABLE CaptureTableYMD
+            (
+                id int,
+                name string,
+                y int,
+                m int,
+                d int
+            )
+            PARTITIONED BY (y,m,d)
```

Comment:
test with y,m,d,h ?

Reply:
This would require setting up another eventhub with different partitioning. I think this is overkill in this test pipeline.